examples/grpc_client/grpc_client_utils.py

```diff
@@ -33,16 +35,22 @@ def prepare_certs(server_cert=None, client_key=None, client_ca=None):
     return server_cert, client_key, client_ca
 
 
-def prepare_stub_and_request(address, model_name, model_version=None, creds=None, opts=None):
+def prepare_stub_and_request(address, model_name, model_version=None, creds=None, opts=None,
+                             get_model_status=False):
     if opts is not None:
         opts = (('grpc.ssl_target_name_override', opts),)
     if creds is not None:
         channel = grpc.secure_channel(address, creds, options=opts)
     else:
         channel = grpc.insecure_channel(address, options=opts)
-
-    stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)
-    request = predict_pb2.PredictRequest()
+    request = None
+    stub = None
+    if get_model_status:
```

Comment:
What do you think about using constants instead? I mean, instead of `get_model_status` flag, we could have `request_type` parameter with PREDICT_REQUEST, MODEL_STATUS_REQUEST etc. I might be easier to extend this fragment in the future, but it's up to you. I'm okay with both ways.

Reply:
it's good idea, i could change `get_model_status` into more generic `request_type` (with values 'inference', 'status', ....)